dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java

```diff
@@ -106,4 +116,216 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         return response;
     }
 
+    @GET
+    @Path ("/sitename/{siteName}/folder/{folder : .+}")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response loadFolderChildrenByURIPath(@Context final HttpServletRequest httpServletRequest,
+                                                      @Context final HttpServletResponse httpServletResponse,
+                                                      @PathParam("siteName") final String siteName,
+                                                      @PathParam("folder") final String uri){
+        Response response = null;
+        final InitDataObject initData = this.webResource.init(null, httpServletRequest, httpServletResponse, true, null);
+        final User user = initData.getUser();
+        try{
+            final String uriParam = !uri.startsWith(StringPool.FORWARD_SLASH) ? StringPool.FORWARD_SLASH.concat(uri) : uri;
+            Host host = APILocator.getHostAPI().findByName(siteName, user, false);
+            //final Folder folder = folderHelper.loadFolderByURI(siteName,user,uriParam);
+            Folder folder = APILocator.getFolderAPI().findFolderByPath(uriParam, host, user, false);
+            if(host==null || folder==null) {
+                throw new Exception("No folder found for "+uri+" on site "+siteName);
+            }
+            CustomFolder root = getFolderStructure(folder, user);
+            response = Response.ok( new ResponseEntityView(root) ).build();
+        } catch (Exception e) { // this is an unknown error, so we report as a 500.
```

Comment:
you do not need to catch anything here, jersey will take care for you

Reply:
I am using the same code that is in the method loadFolderByURI above this one.